src/WebJobs.Script/Rpc/IRpcChannelsManager.cs

```diff
@@ -10,9 +10,7 @@ namespace Microsoft.Azure.WebJobs.Script.Rpc
 {
     public interface IRpcChannelsManager
     {
-        Task InitializeRpcServerAsync();
```

Comment:
Is there a reason why we decided to move this responsibility to the `IRpcChannelsManager` as opposed to just having a single hosted service that initializes both components, but keep their logic in separate services? (e.g. `IRpcChannelsManager` and `IRpcServer`/`IRpcServerManager`)?